tests/models/test_gemini.py

```diff
@@ -709,7 +718,11 @@ async def test_stream_text_no_data(get_gemini_client: GetGeminiClient):
 async def test_stream_structured(get_gemini_client: GetGeminiClient):
     responses = [
         gemini_response(
-            _content_model_response(ModelResponse(parts=[ToolCallPart('final_result', {'response': [1, 2]})])),
```

Comment:
I think all these changes are not necessary since the `tool_call_id` field has a factory.